### PR TITLE
Link transactions page to sending bitcoin page

### PR DESCRIPTION
**Pull request by Bosch-0**
Minor PR adding a link from https://bitcoin.design/guide/payments/transactions/ to https://bitcoin.design/guide/payments/send/

### CODE REVIEW

File: guide/payments/transactions.md [hunk 1]
```diff
@@ -121,3 +121,5 @@ The sender's wallet uses both of these as inputs in the transaction, meaning tha
 Every transaction needs to pay a fee to incentivize miners to include it in a block. There is no fixed fee for making a transaction as it depends on the amount of data it includes, the amount of other transactions that are trying to get verified, and how much each submitter is prepared to pay. Miners typically pick the transactions that will earn them the highest reward to include in a block.
 
 Blocks are also limited in size and new ones are created every 10 minutes on average. This means that if you want a transaction to be confirmed in the next block you might have to pay a relatively high price.
+
```
Comment (johnsBeharry) [inline guide/payments/transactions.md:124]:
```suggestion

---

```

A horizontal rule is typically used to separate this statement from the above content. e.g. https://bitcoin.design/guide/getting-started/visual-language/

**Review by johnsBeharry — CHANGES_REQUESTED** [review]
Nice catch. You getting a bunch of the small details, we need this.

File: guide/payments/transactions.md (hunk 1 above)
Comment (Bosch-0) [reply guide/payments/transactions.md:124]: It's used on all the other page to page links

Comment (johnsBeharry) [reply guide/payments/transactions.md:124]: The text is fine, just need the line spacer. Could you add it in? Should look like this.

Screenshot 2021-05-10 at 13 17 51

**Review by GBKS — APPROVED** [review]

**Review by pavlenex — APPROVED** [review]

**Comment by pavlenex** [comment]
@johnsBeharry waiting for your approval, the spacing issue has been addressed.

**Comment by pavlenex** [comment]
@johnsBeharry 🙏

**Review by johnsBeharry — APPROVED** [review]

**Pull request merged**